subjects/java/checkpoints/reverse-array/README.md

```diff
@@ -0,0 +1,39 @@
+## ReverseArray
+
+### Instructions
+
+In a file named `ReverseArray.java` write a function `reverse` that reverses the given array specified in the parameters and returns it.
```

Comment:
I would rephrase this sentence to improve its clarity

```
Create a Java file named `ReverseArray.java` and define a function named `reverse` inside it. This function should take an array as input, reverse the order of its elements, and return the new reversed array.
```

Reply:
Nice suggestion, Thank you

Reply:
done